mm2src/mm2_core/src/lib.rs

```diff
@@ -19,4 +19,6 @@ impl DbNamespaceId {
         let mut rng = thread_rng();
         DbNamespaceId::Test(rng.gen())
     }
+
+    pub fn for_test_with_id(id: u64) -> DbNamespaceId { DbNamespaceId::Test(id) }
```

Comment:
Could you `inline(always)` this function?

Reply:
I guess this is only used from WASM, if so we can `#[cfg(target_arch = "wasm32")]` it.

Reply:
fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2202/commits/4e46c358ca4d742c3829e327cdd640de17f77e98